sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -0,0 +1,3175 @@
+{
+    "forms": [
+        {
```

Comment:
Can we please remove the forms tag and copy-paste only the array inside it? For example, please check the other formio json.